OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -663,6 +669,8 @@ void trackBulkDownloadSwitchOn(@NonNull String courseId, int totalDownloadableVi
         // Settings events
         String DOWNLOAD_TO_SD_CARD_ON = "Download to sd-card On";
         String DOWNLOAD_TO_SD_CARD_OFF = "Download to sd-card Off";
+        // A/A test
```

Comment:
Same, mention details here, only A/A doesn't not explaining anything

Reply:
done.